docs/trace/tail-based-sampling-span-level/tail-based-sampling-example.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
I don't recall the details, most likely this line (and maybe some other lines) can be removed due to the inheritance (e.g. from global config and folder level config through the folder hierarchy https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/Directory.Build.props).